src/zcl_abapgit_zip.clas.abap

```diff
@@ -7,20 +7,21 @@ CLASS zcl_abapgit_zip DEFINITION
     CLASS-METHODS export
       IMPORTING
         !io_repo       TYPE REF TO zcl_abapgit_repo
+        !iv_logflag    TYPE abap_bool DEFAULT abap_true
```

Comment:
suggest renaming this parameter, it controls if the log should be shown or not

Reply:
Ok, I renamed it to "IV_SHOW_LOG" in the next commit.